suite-native/module-add-coin/src/hooks/useAddCoinAccount.ts

```diff
@@ -0,0 +1,235 @@
+import { useCallback, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { useNavigation } from '@react-navigation/native';
+import { A, pipe } from '@mobily/ts-belt';
+
+import { AccountType, NetworkSymbol, Network } from '@suite-common/wallet-config';
+import {
+    AccountsRootState,
+    DeviceRootState,
+    LIMIT,
+    selectDevice,
+    selectDeviceAccounts,
+} from '@suite-common/wallet-core';
+import { useAlert } from '@suite-native/alerts';
+import {
+    addNetworkAccountThunk,
+    selectAreTestnetsEnabled,
+    selectDiscoverySupportedNetworks,
+    NORMAL_ACCOUNT_TYPE,
+} from '@suite-native/discovery';
+import { useTranslate } from '@suite-native/intl';
+import { useOpenLink } from '@suite-native/link';
+import {
+    RootStackParamList,
+    RootStackRoutes,
+    StackNavigationProps,
+} from '@suite-native/navigation';
+
+const availableNetworkAccountTypes: Record<NetworkSymbol, AccountType[]> = {
```

Comment:
Why is this not derived from wallet-config networks? https://github.com/trezor/trezor-suite/blob/develop/suite-common/wallet-config/src/networksConfig.ts/#L5 I see some discrepancies, for example `test` is supposed to support the same thing as `btc`

Reply:
updated in f69b5332d7